testing/benchmark/outputs.tf

```diff
@@ -43,6 +43,6 @@ output "apm_server_ip" {
 }
 
 output "admin_console_url" {
-  value       = var.run_standalone ? "" : module.ec_deployment[0].admin_console_url
+  value       = var.run_standalone ? "https://cloud.elastic.co/deployments" : module.ec_deployment[0].admin_console_url
```

Comment:
Shouldn't this be `admin.found.no`? If you're running it yourself, then you'll be able to see the deployment in the "normal" (user console) UI. But if it's running in CI, then you won't.

Reply:
I don't think there is a big difference, since it only uses this URL when run against the Moxy when no ESS deployment exists.